package.json

```diff
@@ -61,43 +62,32 @@
     "isparta": "^4.0.0",
     "jsdom": "^11.1.0",
     "lodash.noop": "^3.0.1",
-    "material-ui": "v1.0.0-beta.16",
     "mocha": "^3.5.0",
     "mocha-lcov-reporter": "^1.3.0",
     "nyc": "^11.1.0",
     "prettier": "^1.5.3",
-    "react": "^15.6.1",
-    "react-addons-test-utils": "^15.6.0",
-    "react-dom": "^15.6.1",
+    "react": "^16.3.2",
+    "react-dom": "^16.3.2",
     "react-tap-event-plugin": "^2.0.1",
```

Comment:
I think it's better to remove this, as it is giving warning while installing dependencies:  
```
warning " > react-tap-event-plugin@2.0.1" has incorrect peer dependency "react@^15.4.0-0".
warning " > react-tap-event-plugin@2.0.1" has incorrect peer dependency "react-dom@^15.4.0-0".
```
I asked for advice in material-ui's gitter room and it seems this package is not required any more. See [here](https://gitter.im/mui-org/material-ui?at=5b03ed389ad18967b905d534).

Reply:
I should be removing an and running tests :)